apps/transport/priv/repo/migrations/20230329071947_resource_change_types_datetime_cols.exs

```diff
@@ -0,0 +1,17 @@
+defmodule DB.Repo.Migrations.ResourceChangeTypesDateTimeCols do
+  use Ecto.Migration
+
+  def up do
+    execute "ALTER TABLE resource ALTER COLUMN last_import TYPE timestamp without time zone USING last_import::timestamp;"
```

Comment:
Il n'y a pas contradiction entre `field(:last_import, :utc_datetime_usec)` et `TYPE timestamp without time zone` ?

Reply:
Tous les autres champs en base de données sont ainsi !

https://elixirforum.com/t/why-use-utc-datetime-over-naive-datetime-for-ecto/32532/2 sur la justification de `without time zone`.

Reply:
Je savais pas que le type "timestamp" était interprété comme "timestamp without timezone", du coup je ne m'étais jamais posé la question.

Effectivement si les autres colonnes sont "without timezone" on va laisser comme ça.

C'est quand même intéressant, parce que les liens donnés par le mec qui fait Ecto pointent vers des ressources qui conseillent d'utiliser timestamptz :

- https://www.amberbit.com/blog/2017/8/3/time-zones-in-postgresql-elixir-and-phoenix/
"How can you fix this issue? Use timestamptz data type in PostgreSQL, which is a shortcut to timestamp with time zone."

- https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_timestamp_.28without_time_zone.29

Reply:
compliqué les dates :crying_cat_face: